### PR TITLE
New Topic: replit

### PR DESCRIPTION
**Pull request by frissyn**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Repl.it is a browser-based IDE with a growing community of developers and coders, both beginner and experienced, and more than 50 repositories use this topic.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of your proposed changes.

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by rmathew1992** [comment]
Hi @frissyn  thanks for the contribution ❤️. Unfortunately, as per our contribution guidelines, we don't allow self promotion if you are a maintainer of the project:

> Maintainers of a project cannot add a topic or collection for their own project. If a topic is popular enough to warrant inclusion, someone else will add or improve it.

From your README it looks as though you currently do projects for Repl.it. We do this so that project only get curated if there is enough support/activity in the community. Thank you!

**Comment by frissyn** [comment]
@rmathew1992  I'm not affiliated with nor am I a maintainer of Repl.it, I'm just a community member who makes projects related to it. I understand though. 😔

**Comment by JDOG787** [comment]
Rip :(

**Comment by frionx** [comment]
I have no relation to repl.it, May I be someone who can allow this commit to go through?

**Comment by cursorweb** [comment]
yeah same
oh well

**Review by monstro805 — APPROVED** [review]

**Comment by rmathew1992** [comment]
Oh! In that case, I think this can go through! Sorry about the confusion. I'll re-open this and approve it. Thanks for your contribution!

**Comment by frissyn** [comment]
> Oh! In that case, I think this can go through! Sorry about the confusion. I'll re-open this and approve it. Thanks for your contribution!

Thanks! Sorry if the wording in my README was unclear

File: topics/replit/index.md
Comment (paulcsmith) [inline topics/replit/index.md:2]: Right now CI is failing:

```
topics::replit topic#test_0011_has valid aliases [/home/runner/work/explore/explore/test/topics_test.rb:133]:
alias 'replit' must not be the same as the topic.
Expected "replit" to not be equal to "replit".
```

I believe this can be fixed by removing `replit` from this list.

**Review by paulcsmith — COMMENTED** [review]
Looks like CI is failing. Once it passes I'd be happy to merge this in.

File: topics/replit/index.md
Comment (frissyn) [reply topics/replit/index.md:2]: OK, I'll fix that

**Comment by frissyn** [comment]
Looks like the CI has passed 🎉

**Review by paulcsmith — COMMENTED** [review]
Thanks for your contribution 🥳 👍

**Pull request merged**
